lib/config/sharedConfig.js

```diff
@@ -143,44 +103,6 @@ module.exports = {
         ignoreRegExpLiterals: true,
       },
     ],
-    "no-restricted-imports": [
```

Comment:
these are hardly connected to the project itself, specifically web, I moved to monorepo config